modules/admin/proc-migrating-databases-to-an-external-server.adoc

```diff
@@ -1,11 +1,11 @@
 [id="proc-migrating-databases-to-an-external-server_{context}"]
 = Migrating local databases to an external database server using the Operator
 
-By default, {product} hosts the data for each plugin in a PostgreSQL database. When you fetch the list of databases, you might see multiple databases based on the number of plugins configured in {product-short}. You can migrate the data from an {product-very-short} instance hosted on a local PostgreSQL server to an external PostgreSQL service, such as AWS RDS, Azure database, or Crunchy database. To migrate the data from each {product-very-short} instance, you can use PostgreSQL utilities, such as link:https://www.postgresql.org/docs/current/app-pgdump.html[`pg_dump`] with link:https://www.postgresql.org/docs/current/app-psql.html[`psql`] or link:https://www.pgadmin.org/[`pgAdmin`]. 
+By default, {product} hosts the data for each plugin in a PostgreSQL database. When you fetch the list of databases, you might see multiple databases based on the number of plugins configured in {product-short}. You can migrate the data from an {product-very-short} instance hosted on a local PostgreSQL server to an external PostgreSQL service, such as AWS RDS, Azure database, or Crunchy database. To migrate the data from each {product-very-short} instance, you can use PostgreSQL utilities, such as link:https://www.postgresql.org/docs/current/app-pgdump.html[`pg_dump`] with link:https://www.postgresql.org/docs/current/app-psql.html[`psql`] or link:https://www.pgadmin.org/[`pgAdmin`].
 
 [NOTE]
 ====
-The following procedure uses a link:https://github.com/janus-idp/operator/blob/1.2.x/hack/db_copy.sh[database copy script] to do a quick migration. 
+The following procedure uses a database copy script to do a quick migration.
```

Comment:
This script is part of the documentation itself. So, no need to provide the GitHub Link.